templates/jlink/jlink-builder/jdk-11-buildconfig.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: build.openshift.io/v1
+kind: BuildConfig
+metadata: 
+  name: jlink-builder-jdk-11
+  labels:
+    app: jlink-builder-jdk-11
+spec:
+  source:
+    type: Git
+    git:
+      uri: https://github.com/jmtd/quarkus-quickstarts.git
+    contextDir: getting-started
+    dockerfile: |
+      FROM registry.access.redhat.com/ubi9/openjdk-11 AS ubi9-openjdk-11-jmods
```

Comment:
It should be possible to set `FROM -`, and for OpenShift to supply the correct base image URI. But we would also need to add some more metadata to the buildConfig (see another comment)

Reply:
I've added a basic ubi9 imagestream template in https://github.com/jboss-container-images/openjdk/pull/456,  please take a look, then we can make the suggested change here